src/core/providers/strategies/helpers/signTransactions/signTransactions.ts

```diff
@@ -243,8 +249,24 @@ export async function signTransactions({
       );
       eventBus.subscribe(SignEventsEnum.BACK, onBack);
       eventBus.subscribe(SignEventsEnum.SET_PPU, onSetPpu);
-    };
+    }
+
+    function unsubscribeEvents() {
+      if (!eventBus) {
+        return;
+      }
+
+      eventBus.unsubscribe(SignEventsEnum.NEXT, onNext);
```

Comment:
see pushing to Map technique, then Map foreach unsubscribe. Is used somewhere in the project